src/ifd-nfc.c

```diff
@@ -444,8 +519,12 @@ IFDHGetCapabilities(DWORD Lun, DWORD Tag, PDWORD Length, PUCHAR Value)
       *Value  = 1;
       *Length = 1;
       break;
+#if defined(HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD) && HAVE_DECL_TAG_IFD_POLLING_THREAD
```

Comment:
Please note that your solution is wrong, because
```
#define HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD 0
#ifdef HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD
```
results in `if 1`, which is *not* what we want.